.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: c
+sudo: required
+dist: trusty
+addons:
+  apt:
+    packages:
+      - automake
+      - autotools-dev
+      - git
+      - make
+      - gcc
+      - indent
+      - libncurses5-dev
+      - libslang2-dev
+      - gettext
+      - zlib1g-dev
+      - libselinux1-dev
+      - debhelper
+      - lsb-release
+      - pkg-config
+      - po-debconf
+      - autopoint
+      - libtool
```

Comment:
Can we dockerize (yes, you can set `alias docker=podman` :P) the test env and we can almost get rid of these extra deps?

Reply:
is it something we could do in a different PR?  I realized I wanted a container as well when I needed to install updated software on Travis :-)